warehouse/ingest-core/src/main/java/datawave/formatters/FrequencyColumnValueFormatter.java

```diff
@@ -0,0 +1,58 @@
+package datawave.formatters;
+
+import datawave.query.util.FrequencyFamilyCounter;
+import datawave.query.util.MetadataHelper;
+import org.apache.accumulo.core.data.Key;
+import org.apache.accumulo.core.data.Value;
+import org.apache.accumulo.core.util.format.DefaultFormatter;
+import org.apache.accumulo.core.util.format.Formatter;
+import org.apache.accumulo.core.util.format.FormatterConfig;
+
+import java.util.Comparator;
+import java.util.Iterator;
+import java.util.Map;
+
+public class FrequencyColumnValueFormatter implements Formatter {
+    
+    private Iterator<Map.Entry<Key,Value>> iter;
+    private FormatterConfig config;
+    
+    @Override
+    public boolean hasNext() {
+        return iter.hasNext();
+    }
+    
+    @Override
+    public String next() {
+        StringBuilder sb = new StringBuilder();
+        Map.Entry<Key,Value> entry = iter.next();
+        
+        if (entry.getKey().getColumnQualifier().toString().startsWith(MetadataHelper.COL_QUAL_PREFIX)) {
+            FrequencyFamilyCounter frequencyFamilyCounter = new FrequencyFamilyCounter();
```

Comment:
lets put a try-catch around this section and revert to using the DefaultFormatter in the catch

Reply:
Did a try-finally.   Still have to do Map.Entry<Key,Value> entry = iter.next(); outside of the try block.  Hope that is OK.